src/v2/Apps/Show/__tests__/ShowApp.jest.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { mount } from "enzyme"
+import { ShowApp } from "../ShowApp"
+
+jest.mock("v2/Apps/Show/components/ShowMeta", () => ({
+  ShowMetaFragmentContainer: () => null,
+}))
+
+const SHOW_FIXTURE = { name: "Example Show" }
+
+describe("ShowApp", () => {
+  const getWrapper = () => {
+    return mount(<ShowApp show={SHOW_FIXTURE as any} />)
+  }
```

Comment:
Makes sense, but is this simplified version just a placeholder till the new Relay mocking patterns land in Force?

Reply:
Yeah, or until we need to test something more complex. Like, what are we trying to test really